public_html.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const fs = require("fs");
+const app = express();
+app.listen(3615);
+app.use(express.static('public'));
+app.get("/ha", (req,res)=>{
+  const firstN = req.query.Name;
+  const Name2 = req.query.Comment;
+  fs.writeFile("/home/sya/garageScript/public/public_htmlOut.txt", firstN + " "+Name2);
```

Comment:
Instead of giving req.query.Name a variable just call it in writeFile directly. Try to make your code as simple as possible. Only use variables when it's absolutely necessary.